<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendfiledialog</class>
 <widget class="QWidget" name="sendfiledialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>455</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>455</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SendFile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget
{
	
	background-image: url(:/Client/QT/icon/Background_2.jpg);

}</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;华文隶书&quot;;
color: qlineargradient(spread:repeat, x1:0.72043, y1:0.732955, x2:1, y2:1, stop:0.134409 rgba(0, 0, 0, 255), stop:0.833333 rgba(96, 198, 66, 233));</string>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_message">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>请选择文件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_people">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Comic Sans MS&quot;;
color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_selectfile">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
	border-style:none;
	font: 18pt &quot;华文隶书&quot;;


</string>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
   <property name="icon">
    <iconset resource="Client.qrc">
     <normaloff>:/Client/QT/icon/SlectFile.png</normaloff>:/Client/QT/icon/SlectFile.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_sendfile">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
	border-style:none;
	font: 18pt &quot;华文隶书&quot;;


</string>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
   <property name="icon">
    <iconset resource="Client.qrc">
     <normaloff>:/Client/QT/icon/SendFile_blue.png</normaloff>:/Client/QT/icon/SendFile_blue.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="Client.qrc">
     <normaloff>:/Client/QT/icon/talk.png</normaloff>:/Client/QT/icon/talk.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>340</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>455</width>
     <height>340</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>455</width>
     <height>340</height>
    </size>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>label_title</zorder>
  <zorder>label_message</zorder>
  <zorder>label_people</zorder>
  <zorder>pushButton_selectfile</zorder>
  <zorder>pushButton_sendfile</zorder>
  <zorder>toolButton</zorder>
 </widget>
 <resources>
  <include location="Client.qrc"/>
 </resources>
 <connections/>
</ui>
